eclipse-scout-core/src/desktop/outline/pages/PageTileButton.ts

```diff
@@ -24,18 +24,39 @@ export class PageTileButton extends TileButton implements PageTileButtonModel {
   protected override _init(model: InitModelOf<this>) {
     super._init(model);
 
-    this.label = this.page.text;
-    this.iconId = this.page.overviewIconId;
-    this.labelHtmlEnabled = this.page.htmlEnabled;
+    this.label = this._text;
+    this.iconId = this._iconId;
+    this.labelHtmlEnabled = this._htmlEnabled;
 
     this.on('click', (event: Event<Button>) => {
       this.outline.selectNode(this.page);
     });
   }
 
   notifyPageChanged() {
-    this.setLabel(this.page.text);
-    this.setIconId(this.page.overviewIconId);
-    this.setLabelHtmlEnabled(this.page.htmlEnabled);
+    this.setLabel(this._text);
+    this.setIconId(this._iconId);
+    this.setLabelHtmlEnabled(this._htmlEnabled);
+  }
+
+  protected get _text(): string {
+    if (this.page?.overviewText !== undefined) {
+      return this.page.overviewText;
+    }
+    return this.page?.text;
+  }
+
+  protected get _iconId(): string {
+    if (this.page?.overviewIconId !== undefined) {
+      return this.page.overviewIconId;
+    }
+    return this.page?.iconId;
```

Comment:
This changes the behavior for Java pages. It is not possible anymore to _not_ have an overviewIcon. Maybe set overviewIcon to null in OutlineAdapter?